contracts/StorageManager.sol

```diff
@@ -192,16 +234,15 @@ contract StorageManager {
 
         // Allow to enforce payout funds and close of agreements that are already expired,
         // which should free needed capacity, if the capacity is becoming depleted.
-        if (agreementsReferencesToBePayedOut.length > 0) {
-            _payoutFunds(agreementsReferencesToBePayedOut, payable(provider));
+        if (dataReferencesOfAgreementToPayOut.length > 0) {
+            _payoutFunds(dataReferencesOfAgreementToPayOut, creatorsOfAgreementToPayOut, tokenOfAgreementsToPayOut, payable(provider));
```

Comment:
Idea: because we are only allowing payout of one token. Perhaps we should have a public function which does this as well, so we can still force payout of *all* agreements which are expired.

Reply:
Done: This was surprisingly easy to do. We add param provider to the public function payoutFunds.
@julianmrodri , this is a new feature, essentially to allow people to free capacity from expired agreements. We do this also in the function `newAgreement`. However, compared to the previous version, newAgreement only allows to do a payout for a single token. If you would like to force payout of expired agreements otherwise, now this is possible via this public function.